packages/language-server/src/plugins/typescript/languageService.ts

```diff
@@ -55,12 +55,33 @@ async function createLanguageService(tsconfigPath: string, workspaceRoot: string
     },
   };
 
-  const configJson = getDefaultJsConfig();
+  let configJson = (tsconfigPath && ts.readConfigFile(tsconfigPath, ts.sys.readFile).config) || {};
+
+  // If our user has types in their config but it doesn't include the types for ImportMeta, let's add them for them
+  if (
+    configJson.compilerOptions?.types &&
+    !configJson.compilerOptions?.types.includes("vite/client")
+  ) {
+    configJson.compilerOptions.types.push("vite/client");
+  }
+
+  configJson.compilerOptions = Object.assign(
```

Comment:
Using `Object.assign` mean that this is a shallow merge, I'm not sure if that could cause issues but I don't think so?

Reply:
Nope, should be ok